ProcessMaker/processmaker
# Bugfix/FOUR-4840: List of Activities does not mention conditional Event

## Issue & Reproduction Steps
When you have a process with an Intermediate Conditional Catch Event message for that event is not showed in list of activities for request.
- Create a process with an Intermediate Conditional Catch Event.
- Create new request and then go to Requests > Select the request and take a look at the list of activities, you should not see a message for the conditional event.

## Solution
- Added message for Conditional Event Definition Interface.
- Added missing translations

## How to Test
- Create a process with an Intermediate Conditional Catch Event.
- Create new request and then go to Requests > Select the request and take a look at the list of activities, you should see a message for the conditional event.

**Sample process**
[Verify Event Based Gateway Conditional and Message Event.json.zip](https://github.com/ProcessMaker/processmaker/files/7677380/Verify.Event.Based.Gateway.Conditional.and.Message.Event.json.zip)

**Working video**

https://user-images.githubusercontent.com/90727999/145242930-5d9a1d39-9180-4d6b-9ffa-10fde2f220b9.mov


## Related Tickets & Packages
- [FOUR-4840](https://processmaker.atlassian.net/browse/FOUR-4840)

## Code Review Checklist
